packages/client/src/arpa_reporter/views/Home.vue

```diff
@@ -129,6 +136,34 @@ export default {
       const periodId = this.$store.getters.viewPeriod.id || 0;
       return apiURL(`/api/exports?period_id=${periodId}`);
     },
+    async sendTreasuryReport() {
+      this.sending = true;
+
+      try {
+        const result = await getJson('/api/exports?async=true');
+
+        if (result.error) {
+          this.alert = {
+            text: 'Something went wrong. Unable to send an email containing the treasury report. Reach out to grants-helpdesk@usdigitalresponse.org if this happens again.',
+            level: 'err',
+          };
+          console.log(result.error);
+        } else {
+          this.alert = {
+            text: 'Sent. Please note, it could take up to 1 hour for this email to arrive.',
```

Comment:
Small nit, and a bit outside my wheelhouse, but I'd suggest tweaking this wording to focus on "we received/are working on your request" rather than "sent", since the latter is a bit ambiguous wrt to the email aspect.

Reply:
@TylerHendrickson I'm going to keep this as-is for now given that it is only visible for one-partner. Will incorporate the final copy when I flip the feature-flag.